front/app/components/IdeaCard/Footer/ReadMoreButton.tsx

```diff
@@ -6,16 +6,14 @@ import messages from '../messages';
 import { useIntl } from 'utils/cl-intl';
 
 interface Props {
-  slug?: string;
   onClick?: () => void;
```

Comment:
This component is used in only 1 place, and it was not passing a slug parameter. So we're safe to remove it :)